tools/openms/FeatureFinderCentroided.xml

```diff
@@ -64,57 +63,57 @@ python3 '$__tool_directory__/fill_ctd.py' '@EXECUTABLE@.ctd' '$args_json' '$hard
       </section>
       <section name="fit" title="Settings for the model fitting" help="" expanded="false">
-        <param name="max_iterations" argument="-algorithm:fit:max_iterations" type="integer" optional="true" min="1" value="500" label="Maximum number of iterations of the fit" help=""/>
+        <param name="max_iterations" argument="-algorithm:fit:max_iterations" type="integer" optional="false" min="1" value="500" label="Maximum number of iterations of the fit" help=""/>
       </section>
       <section name="feature" title="Settings for the features (intensity, quality assessment, ...)" help="" expanded="false">
-        <param name="min_score" argument="-algorithm:feature:min_score" type="float" optional="true" min="0.0" max="1.0" value="0.7" label="Feature score threshold for a feature to be reported" help="The feature score is the geometric mean of the average relative deviation and the correlation between the model and the observed peaks"/>
-        <param name="min_isotope_fit" argument="-algorithm:feature:min_isotope_fit" type="float" optional="true" min="0.0" max="1.0" value="0.8" label="Minimum isotope fit of the feature before model fitting" help=""/>
-        <param name="min_trace_score" argument="-algorithm:feature:min_trace_score" type="float" optional="true" min="0.0" max="1.0" value="0.5" label="Trace score threshold" help="Traces below this threshold are removed after the model fitting.. This parameter is important for features that overlap in m/z dimension"/>
-        <param name="min_rt_span" argument="-algorithm:feature:min_rt_span" type="float" optional="true" min="0.0" max="1.0" value="0.333" label="Minimum RT span in relation to extended area that has to remain after model fitting" help=""/>
-        <param name="max_rt_span" argument="-algorithm:feature:max_rt_span" type="float" optional="true" min="0.5" value="2.5" label="Maximum RT span in relation to extended area that the model is allowed to have" help=""/>
-        <param name="rt_shape" argument="-algorithm:feature:rt_shape" type="select" optional="true" label="Choose model used for RT profile fitting" help="If set to symmetric a gauss shape is used, in case of asymmetric an EGH shape is used">
+        <param name="min_score" argument="-algorithm:feature:min_score" type="float" optional="false" min="0.0" max="1.0" value="0.7" label="Feature score threshold for a feature to be reported" help="The feature score is the geometric mean of the average relative deviation and the correlation between the model and the observed peaks"/>
```

Comment:
optional=false we can also remove everywhere.

Reply:
Will try to remove `optional` if not needed. Not entirely trivial since: 

```
Defaults to false except when the type attribute value is select and multiple is true.
```

But a good idea anyway.

Reply:
Fixed in https://github.com/galaxyproteomics/tools-galaxyp/pull/697/commits/fd8160798d3a3e03783467ecb0bf1cfc8e71ddf2